ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -931,4 +931,154 @@ spec:
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Block (default)'':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Windows Defender Firewall with Advanced Security\Windows Defender Firewall with Advanced Security - Local Group Policy Object\Windows Defender Firewall Properties\Private Profile\Inbound connections'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\WindowsFirewall\\PrivateProfile\DefaultInboundAction' and data == 1);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_9.2.2
+  contributors: RachelElysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Windows Firewall: Private: Outbound connections' is set to 'Allow (default)' (Automated)
+  platforms: win10
+  platform: windows
+  description: |
+    This setting determines the behavior for outbound connections that do not match an outbound firewall rule.
+    The recommended state for this setting is: Allow (default).
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Allow (default)'':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Windows Defender Firewall with Advanced Security\Windows Defender Firewall with Advanced Security - Local Group Policy Object\Windows Defender Firewall Properties\Private Profile\Outbound connections'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\WindowsFirewall\\PrivateProfile\DefaultOutboundAction' and data == 0);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_9.2.3
+  contributors: RachelElysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Windows Firewall: Private: Settings: Display a notification' is set to 'No' (Automated)
+  platforms: win10
+  platform: windows
+  description: |
+    Select this option to have Windows Firewall with Advanced Security display notifications to the user when a program is blocked from receiving inbound connections.
+    The recommended state for this setting is: No.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to No:
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Windows Defender Firewall with Advanced Security\Windows Defender Firewall with Advanced Security - Local Group Policy Object\Windows Defender Firewall Properties\Private Profile\Settings Customize\Display a notification'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\WindowsFirewall\\PrivateProfile\DisableNotifications' and data == 0);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_9.2.4
+  contributors: RachelElysia
+---
+# apiVersion: v1
```

Comment:
Policies should not be commented out. You can just put `TBD` on the query section and  the following tag to the tags section: `CIS_not_completed`

Reply:
Good to know! Thanks @marcosd4h!